src/features/targeting/hocs/withTargetedClickThrough.js

```diff
@@ -41,7 +41,7 @@ function withTargetedClickThrough<Config>(
         return (
             <WrappedComponent showCloseButton stopBubble {...rest} isShown={shouldShow} onDismiss={onClose}>
                 <span
-                    className="bdl-targeted-click-through"
+                    className="bdl-targeted-click-through bdl-ButtonWrapper"
```

Comment:
Hi @alexkrolick - I realized the element I want to add the class to is in BUIE. How do you feel about adding the class directly here? Or do you think it's better to pass it as a prop? I think we would want all cases of this element to have  .bdl-ButtonWrapper so I think adding it directly makes sense. What do you think?

Reply:
Makes sense to add it here I think

Reply:
* Assuming that it always or nearly always wraps a button

Reply:
It seems a little unsafe to make such a big assumption, since the tooltip could wrap almost any kind of element. Given that, should we revert back to the original solution, with `.bdl-targeted-click-through > .btn` in the ButtonGroup styles?

Reply:
Yes I agree with both you, it's not safe to assume it's always wraps a button, I will change it back